gene/schemas.py

```diff
@@ -11,6 +11,30 @@
 import re
 
 
+def check_curie(cls, v):
```

Comment:
If someday we end up reusing these models in a wider context, we could think about doing more robust enforcement of CURIE character standards:

- prefix must be an NCName (https://stackoverflow.com/a/6159003)
- identifier must be an "irelative-ref" per the IRI spec but it's not clear to me what that means

Reply:
- We do our validation for prefixes already when using `NamespacePrefix` during ETL methods.
- `irelative-ref`is the grammar shown here: https://www.ietf.org/rfc/rfc3987.txt